lib/assessment.js

```diff
@@ -4,31 +4,62 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(nums) {
+  //grab the first number to compare against
+  var minValue = nums[0];
+  // loop through the remaining numbers
+  for(var i = 1; i < nums.length; i++){
+    if(nums[i] < minValue){
+      //replace minValue with new number
+      minValue = nums[i];
+    }
+    return minValue;
+  }
 };
 
+
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
 
+var sum = function sum(nums) {
+  var sumValue = 0;
+  for(var i = 0; i < nums.length; i++){
+      sumValue += nums[i];
+  }
+return sumValue;
 };
 
+
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  //create a new empty array
+  //loop through length and push default Value to the new empty arrays
+    var Array = [];
```

Comment:
`Array` is a built-in constructor function the interpreter provides. It will not allow you to create a variable with that name. It will throw an error with a very strange message.